### PR TITLE
Add pra17dod's Project Report "Microfarm For Everybody"

### LINKED ISSUE #801: Add Project Report for "Microfarm For Everybody" (Django Microfarm)
Add Project Report for "Microfarm For Everybody" (Django Microfarm) 
which is taken up in GSoC' 2021

### PR DESCRIPTION
**Pull request by pra17dod**
This will add the Project Report for "Microfarm for Everybody"
A.K.A "Django Microfarm" done by pra17dod during the GSoC' 2021
period.

Closes https://github.com/coala/projects/issues/801

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by pra17dod** [comment]
@abhishalya @KVGarg Please review the PR

**Comment by abhishalya** [comment]
@pra17dod can you fix the commit message, will fix the travis build here.

File: _reports/pra17dod.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+  - Work Report
+date: 2021-08-22
+student: Prashant Dodiya
+organisation: coala
+organisation_link: https://coala.io
+project: Microfarm For Everybody
+project_link: https://summerofcode.withgoogle.com/projects/#6093515215339520
+mentors: >
+  [Finn Peters](https://github.com/finn-1),
+  [Lasse Peters](),
```
Comment (abhishalya) [inline _reports/pra17dod.md:14]: You can omit this entry.

**Review by abhishalya — CHANGES_REQUESTED** [review]

File: _reports/pra17dod.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+      - repo: GSoC/GSoC-2021
+      - link: https://gitlab.com/coala/GSoC/gsoc-2021/-/issues/1
+      - details: >
+          Deciding project requirements and database handling
+
+   - 3:
+      - repo: GSoC/GSoC-2021
+      - link: https://gitlab.com/coala/GSoC/gsoc-2021/-/issues/52
+      - details: >
+          Blog for Community Bonding Period
```
[the remark this answers is not in the document]
Comment (pra17dod) [reply _reports/pra17dod.md:68]: Yaa right!

File: _reports/pra17dod.md (hunk 1 above)
Comment (pra17dod) [reply _reports/pra17dod.md:14]: Oky

**Review by abhishalya — APPROVED** [review]
LGTM, we can have updates later if there's something to change/add.

cc @bkhanale @KVGarg

**Review by bkhanale — APPROVED** [review]
Yeah, merging this for now. Please raise another PR for updates.

**Pull request merged**